.jenkins/pytorch/macos-build.sh

```diff
@@ -14,6 +14,7 @@ if [ ! -d "${PYTORCH_ENV_DIR}/miniconda3" ]; then
 fi
 export PATH="${PYTORCH_ENV_DIR}/miniconda3/bin:$PATH"
 source ${PYTORCH_ENV_DIR}/miniconda3/bin/activate
+conda install python=3.6
```

Comment:
This looks pretty strange. Are you... downgrading to python 3.6? When can we upgrade to Python 3.7? What exactly goes wrong without it?

Reply:
I just tested with Python 3.7 and it seems to work pretty well :)